src/shared/bus-polkit.c

```diff
@@ -245,6 +245,47 @@ static AsyncPolkitQuery *async_polkit_query_free(AsyncPolkitQuery *q) {
 
 DEFINE_TRIVIAL_CLEANUP_FUNC(AsyncPolkitQuery*, async_polkit_query_free);
 
+static int async_polkit_query_new(
+                AsyncPolkitQueries *parent,
+                sd_bus_message *request,
+                const char *action,
+                const char **details,
+                AsyncPolkitQuery **ret)
+{
```

Comment:
Coding style: please do not insert line-break between `)` and `{`.

Reply:
Done.